src/styles/blocks/stars.css

```
@@ -0,0 +1,21 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(../../images/star.svg);
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
```

у последней звездочки не должно было быть правого марджина
```suggestion
.stars .stars__star:not(:last-child) {
  margin-right: 4px;```